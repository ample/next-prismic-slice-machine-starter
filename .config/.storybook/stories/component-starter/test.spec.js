// TODO: REMOVE the "FPO snapshot test" below and add additional tests.
// ----  Snapshots tests are not necessary because theyâ€™re created automatically with Storyshots.
// ---------------------------------------------------------
// ----  If tests are not required for the component, remove this file.
// ---------------------------------------------------------

import renderer from 'react-test-renderer';

// ---------------------------------------------------------

import { component as ComponentName, fixtures } from '.';

// ---------------------------------------------------------

describe('ComponentName', () => {
  it('FPO snapshot', () => {
    const tree = renderer
      .create(<ComponentName {...fixtures.componentProps} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
