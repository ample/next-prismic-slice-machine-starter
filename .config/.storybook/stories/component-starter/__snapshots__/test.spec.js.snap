// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ComponentName FPO snapshot 1`] = `
<a
  className="component_name"
  href="url"
>
  <h2>
    name of prop content goes here.
  </h2>
</a>
`;
